src/megachatapi_impl.cpp

```diff
@@ -6077,7 +6454,8 @@ void MegaChatSessionHandler::onRemoteStreamRemoved()
 void MegaChatSessionHandler::onPeerMute(karere::AvFlags av, karere::AvFlags oldAv)
 {
     MegaChatCallPrivate* chatCall = callHandler->getMegaChatCall();
-    chatCall->setRemoteAudioVideoFlags(av);
+    megaChatSession->setAvFlags(session->receivedAv());
```

Comment:
Should we use here the received `av` in the params of the callback, rather than the ones from the `session`??

Reply:
`av` and `session->receivedAv()` have the same values. It is possible use both

Reply:
OK, let's use then the received value in the params for readability.